Sources_v3/Query/ChannelQuery.swift

```diff
@@ -28,9 +28,10 @@ public struct ChannelQuery<ExtraData: ExtraDataTypes>: Encodable {
     /// ChannelCreatePayload that is needed only when creating channel
     let channelPayload: ChannelEditDetailPayload<ExtraData>?
     
-    /// ChannelId this query handles.
-    var cid: ChannelId {
-        guard let id = id else { return .init(type: type, id: "") }
+    /// `ChannelId` this query handles.
+    /// If `id` part is missing then it's impossible to create valid `ChannelId`.
+    public var cid: ChannelId? {
```

Comment:
nit: Here you're changing code you introduced in 000802cfcd718fbced3850c9abb7e9756d8feca5. It makes the review more difficult if each line is not changed exactly once.

Reply:
Yep, sorry for that.
Mixed up with `fixups`.